hunts/src/puzzlesSlice.js

```diff
@@ -83,6 +83,15 @@ export const addPuzzleTag = createAsyncThunk(
   }
 );
 
+function priority(row) {
```

Comment:
Can you scope this inside puzzleComparator()?

Also, for consistency could you you return 1 for Hi-pri and -1 for low-pri, and return `priority(b)-priority(a)`?